summary.proto

```diff
@@ -48,4 +53,12 @@ message Summary {
   }
 
   repeated PostgresqlVersion postgresql_version = 6;
+
+  message RbacStats {
```

Comment:
This also needs to be defined in the top-level message also?

Reply:
You mean used? Yes!